css/styles1.css

```diff
@@ -0,0 +1,501 @@
+* {
+  margin: 0;
+  padding: 0;
+
+  /* color: var(--lightBlack); */
+  font-family: 'Lato', sans-serif;
+  scroll-behavior: smooth;
+}
+
+.nav,
+.intro-text {
+  background-image: url("../Images/nav-bg.png");
+}
+
+.nav-social,
+.nav-bar,
+.partners-desktop {
+  display: none;
+}
+
+.nav-ham-icon {
+  display: inline-block;
+  cursor: pointer;
+  margin-left: 20px;
+  margin-top: 20px;
+  height: 50px;
+}
+
+.bar {
+  width: 25px;
+  height: 5px;
+  background-color: #333;
+  margin: 4px 0;
+  transition: 0.4s;
+}
+
+.bar1.pressed {
+  transform: translate(0, 11px) rotate(-45deg);
+  background-color: white;
+}
+
+.bar2.pressed { opacity: 0; }
+
+.bar3.pressed {
+  transform: translate(0, -11px) rotate(45deg);
+  background-color: white;
+}
+
+.intro-text > h3,
+h1 {
+  padding: 0 20px;
+  color: #ec5242;
+
+  /* font-size: 20; */
+}
+
+.intro-text > h1 {
+  font-weight: 700;
+  font-size: 36px;
+  display: block;
+  padding: 0 40px;
+}
+
+.intro-text > h3 {
+  display: inline-block;
```

Comment:
- [ ] Kindly make sure your texts are bigger, easier to read, and have a similar `font-size` to the original design 🙏 

Original Design | Your Desktop Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/203814723-daf101f7-1384-4496-8c06-236dd82e87d7.png)  |  ![image](https://user-images.githubusercontent.com/74814780/203814697-b07adfc5-c91d-4757-9a71-1b1f22e8533a.png)